libpromises/failsafe.cf

```diff
@@ -114,6 +118,12 @@ bundle agent failsafe_cfe_internal_update
         depth_search => failsafe_recurse("inf"),
         classes => failsafe_results("namespace", "inputdir_update");
 
+      "$(sys.workdir)/modules"
+        handle => "failsafe_cfe_internal_bootstrap_update_files_sys_workdir_modules_shortcut",
```

Comment:
I copied this from the `inputdir` section. I have no opinion on what it should be. Maybe @nickanderson does?